spec/views/transactions/edit.html.erb_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe 'Edit Transaction', type: :system do
+  describe 'edit page' do
+    it 'shows the right content' do
+      visit edit_category_transaction_path(1, 1)
+      expect(page).to have_content('You need to sign in')
+    end
+  end
+end
```

Comment:
- [ ] [OPTIONAL] While I seem to have missed this when requesting you to make the other changes, it would be great if you also tested the transaction index page.